### ISSUE TITLE
Tensorboard only showing learning rate and strange Loss

### ISSUE THREAD

**Issue by snoopypi** [tensorflow/models#8001]
Hi there
I trained several times and now have a lot of events.out.tfevents- files.
Now  I would like to analyze these data in tensorboard because I need the accuracy/mAP of the different training sessions.

Problem now if I open tensorboard with  my last training session:
tensorboard --logdir="./traing"  

I only see learning rate and a (strange) Loss. (see printscreen)

How can i see more? Especially the accuracy and so?

I have tensorflow/board 1.13.1, Ubuntu 18.04

Thank you
![Screenshot from 2020-01-04 10-53-34](https://user-images.githubusercontent.com/48096442/71763866-519d3d00-2ee1-11ea-80e4-14f2a1ca37c6.png)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by snoopypi** [comment]
Hi well I trained on my host. :)

Have I written custom code No
OS Platform and Distribution Ubuntu 18.04
TensorFlow installed from pip3 install
TensorFlow version 1.13.1
Bazel version N/A
CUDA/cuDNN version N/A
GPU model and memory N/A

I followed this tutorial
https://tensorflow-object-detection-api-tutorial.readthedocs.io/en/latest/training.html

Thanks

**Comment by snoopypi** [comment]
Does noone have a solution for this?
I also tried to run eval.py after training and now I can see some more things but still not seeing mAP!

**Comment by snoopypi** [comment]
Does no one have a solution for this?
I also tried to run eval.py from /legacy and now seeing some more things but still not mAP! 

Thank you!
